toolkits/golang.go

```diff
@@ -334,6 +334,23 @@ func (toolkit GoToolkit) PrepareForStepRun(step stepmanModels.StepModel, sIDData
 	return goBuildStep(&defaultRunner{}, goConfig, step.Toolkit.Go.PackageName, stepAbsDirPath, fullStepBinPath)
 }
 
+func GoBuildStep(stepAbsDirPath string, packageName string, fullStepBinPath string) error {
```

Comment:
nit: I know we probably won't introduce new toolkits, but I still think this method should be part of the toolkit interface. Offline mode is now a CLI-wide concept, so every toolkit should support it